xbmc/guilib/WindowIDs.dox

```diff
@@ -142,7 +146,15 @@ This page shows the window names, the window definition, the window ID and the s
 
 \section window_ids_rm Additional revision history
 
+\subsection modules_rm_windowids_v21 Kodi v20 (Nexus)
```

Comment:
This section is for removals, additions are already identified on the table above

Reply:
K, removing